app/containers/ViewQuestion/BountyPopover.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { memo } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import {
+  BG_LIGHT,
+  BORDER_PRIMARY_LIGHT,
+  BORDER_RADIUS_L,
+  TEXT_DARK,
+} from 'style-constants';
+import { FormattedMessage } from 'react-intl';
```

Comment:
Import elements in groups. Use one logic: libraries separately, constants separately, messages, utils, selectors etc.